tests/isolated_testcase.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import os
+
+from errbot.backends.test import FullStackTest
+from pathlib import Path
+
+
+class IsolatedTestCase(FullStackTest):
+
+    def setUp(self, extra_config=None):
+        file_name = 'labhub.plug'
+        file_rename = 'hidden'
```

Comment:
do class initialisation in [`setUpClass`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass) , including creating the hidden directory , and then remove the directory in `tearDownClass`.